src/audio/src/src.c

```diff
@@ -709,9 +709,9 @@ static int src_get_copy_limits(struct comp_data *cd,
 	}
 
 	if (sp->blk_in == 0 && sp->blk_out == 0)
-		return -EIO;
+		return false;
 
-	return 0;
+	return true;
```

Comment:
Thanks, it looks nice, would further suggest use a separate patch for this change.